app/presenters/tree_builder.rb

```diff
@@ -257,9 +257,15 @@ def x_build_node(object, pid, options)
     node = x_build_single_node(object, pid, options)
 
     # Process the node's children
+    load_children = if object.kind_of?(Struct)
```

Comment:
Can we just use hash here, like every other tree?

We are in the process of getting rid of hash nodes, but we should be doing that systematically for all the trees... This looks too much like an exception for a signle tree...